src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
```

Comment:
Lacking a header comment here. An example could be : Represents an adding or removing a Tag to/from a Person